_data/security.yml

```diff
@@ -49,6 +49,16 @@ websites:
       tfa: No
       twitter: gogetssl
 
+    - name: Imperva Incapsula
+      url: http://www.Incapsula.com/
+      img: incapsula.png
+      tfa: Yes
+      email: Yes
+      sms: Yes
+      software: Yes
+      doc: https://www.incapsula.com/website-security/two-factor-authentication.html
```

Comment:
Change this line to:
`doc: https://docs.incapsula.com/Content/Management%20Console%20and%20Settings/accountsettings.htm?Highlight=Factor`